enterprise/frontend/src/embedding-sdk/components/public/DashboardCreateModal/DashboardCreateModal.tsx

```diff
@@ -0,0 +1,23 @@
+import Modal from "metabase/components/Modal";
+import { CreateDashboardModalConnected } from "metabase/dashboard/containers/CreateDashboardModal";
+import type { CollectionId, Dashboard } from "metabase-types/api";
+
+export interface DashboardCreateModalProps {
+  collectionId?: CollectionId | null;
+  onCreate: (dashboard: Dashboard) => void;
+  onClose?: () => void;
+}
+
+export const DashboardCreateModal = (props: DashboardCreateModalProps) => {
+  const { collectionId, onCreate, onClose } = props;
+
+  return (
+    <Modal onClose={onClose}>
+      <CreateDashboardModalConnected
+        onCreate={onCreate}
+        onClose={onClose}
+        collectionId={collectionId}
```

Comment:
I'm not sure this prop does anything since I tried passing a collection ID, but the collection picker is still showing the root collection by default.
![image](https://github.com/user-attachments/assets/66ef89a4-d5d2-4f1c-bd92-f92604c532ca)


I'd expect this prop to behave similar to the collectionId we passed to the collection browser, so that I could limit users from seeing every instance collections.

Reply:
This seems to be working now, but it's different from the collection browser's `collectionId` which limits the collections that users can see. This is just changing the starting collection. I think this feature needs to be made on the collection browser itself to support this.

I still think it should be have similarly to our collection browser`s `collectionId` for consistency.